src/VisualStudio/Core/Def/Telemetry/VisualStudioWorkspaceTelemetryService.cs

```diff
@@ -60,10 +62,46 @@ protected override void TelemetrySessionInitialized()
                 var logDelta = _globalOptions.GetOption(DiagnosticOptionsStorage.LogTelemetryForBackgroundAnalyzerExecution);
 
                 // initialize session in the remote service
-                _ = await client.TryInvokeAsync<IRemoteProcessTelemetryService>(
+                var remoteProcessId = await client.TryInvokeAsync<IRemoteProcessTelemetryService, int>(
                     (service, cancellationToken) => service.InitializeTelemetrySessionAsync(Process.GetCurrentProcess().Id, settings, logDelta, cancellationToken),
                     CancellationToken.None).ConfigureAwait(false);
+
+                HookRemoteProcessExitCodeLogger(remoteProcessId);
             });
         }
+
+        private static void HookRemoteProcessExitCodeLogger(Optional<int> remoteProcessId)
+        {
+            if (!remoteProcessId.HasValue)
+            {
+                FaultReporter.LogError($"Roslyn telemetry service initialization failed.");
```

Comment:
where do tehse get reported to us through?  (curious how there's no FunctionId for this).

Reply:
ServiceHub log.